bin/hipconfig

```diff
@@ -87,11 +87,11 @@ $HSA_PATH=$ENV{'HSA_PATH'} // "$ROCM_PATH/hsa";
 $HIP_CLANG_PATH=$ENV{'HIP_CLANG_PATH'} // "$ROCM_PATH/llvm/bin";
 
 #---
-#HIP_PLATFORM controls whether to use NVCC or HCC for compilation:
-$HIP_PLATFORM=$ENV{'HIP_PLATFORM'};
 # Read .hipInfo
 my %hipInfo = ();
 parse_config_file("$HIP_PATH/lib/.hipInfo", \%hipInfo);
+#HIP_PLATFORM controls whether to use NVCC or HCC for compilation:
+$HIP_PLATFORM = $hipInfo{'HIP_PLATFORM'} // "hcc";
 $HIP_COMPILER = $hipInfo{'HIP_COMPILER'} // "hcc";
 $HIP_RUNTIME = $hipInfo{'HIP_RUNTIME'} // "HCC";
```

Comment:
A misleading annotation has to be removed.

Reply:
Ah yeah, that should be removed. Just remove it

Reply:
I meant all 'hcc' as well.

Reply:
Just removed 'hcc' as the fallback option.